LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-invoker-reset-expected.txt

```diff
@@ -0,0 +1,4 @@
+Popover 1  Button Popover 2
+
+FAIL Invoker gets reset appropriately assert_false: invoker was not used to show p2, so p1 should close expected false got true
```

Comment:
Probably https://github.com/WebKit/WebKit/pull/13152 needs to land first to fix this FAIL.